opal/mca/common/ucx/common_ucx_wpool.c

```diff
@@ -31,6 +31,8 @@ __thread FILE *tls_pf = NULL;
 __thread int initialized = 0;
 #endif
 
+bool mpi_thread_multiple_enabled = true;
```

Comment:
This should be either `static` inside a single TU or properly prefixed (`opal_mca_common_ucx_` in this case)

Reply:
done